src/bucket.js

```diff
@@ -2099,6 +2099,9 @@ Bucket.prototype.setUserProject = function(userProject) {
  *     MD5 checksum for maximum reliability. CRC32c will provide better
  *     performance with less reliability. You may also choose to skip validation
  *     completely, however this is **not recommended**.
+ * @param {object} [options.requestOptions] When `pathString` is an URL additional
+ *     options for HTTP request could be provided here.
+ * @see [`request` options list](https://github.com/request/request#requestoptions-callback)
```

Comment:
@callmehiphop are links still supported within a description? In other words:

... [options.requestOptions] When `pathString` is ...
... [options for HTTP request](http://)

Reply:
@stephenplusplus There are similar cases of links usage in this file, so I updated my PR.